packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -33,7 +35,14 @@ const _removeGranuleFromCmr = async (granule, collectionId) => {
 
   const cmrSettings = await cmrjsCmrUtils.getCmrSettings();
   const cmr = new CMR(cmrSettings);
-  const metadata = await cmr.getGranuleMetadata(granule.cmr_link);
+  try {
+    metadata = await cmr.getGranuleMetadata(granule.cmr_link);
+  } catch (error) {
+    if (error instanceof GranuleNotPublished) {
+      log.warn(`Granule ${granule.granule_id} in Collection ${collectionId} does not have a CMR link, so its metadata cannot be retrieved`);
```

Comment:
```suggestion
      log.warn(`Granule ${granule.granule_id} in Collection ${collectionId} does not have a CMR link, so its metadata cannot be retrieved`);
      return;
```

Reply:
@Nnaga1 , sorry to not suggest sooner but you want to return so that the error isn't thrown in the next line